### PR TITLE
Document ams model

### PR DESCRIPTION
**Pull request by aldreth**
References https://github.com/rails-api/active_model_serializers/issues/1283

I think this resolves 2 of your checkboxes - https://github.com/rails-api/active_model_serializers/pull/1272#discussion_r42111785 & https://github.com/rails-api/active_model_serializers/pull/1272#discussion_r42111871

I don’t know what to do to resolve https://github.com/rails-api/active_model_serializers/pull/1272#discussion_r42172925 & don’t know what to put in the changelog.

### CODE REVIEW

**Comment by bf4** [comment]
One big problem/bug right now with ams::model is that the attributes hash doesn't sync with the attr_accessor values.  This causes all sorts of problems.  It's essentially and unresolved implementation detail between ActiveModel::Model and ActiveModel::Serialization.

File: test/cache_test.rb [hunk 1]
```diff
@@ -250,8 +250,9 @@ def test_a_serializer_rendered_by_two_adapter_returns_differently_fetch_attribut
       # Assert attributes are serialized correctly
       serializable_alert = serializable(alert, serializer: AlertSerializer, adapter: :attributes)
       attributes_serialization = serializable_alert.as_json
-      assert_equal expected_fetch_attributes, alert.attributes
-      assert_equal alert.attributes, attributes_serialization
+      alert_attributes_symbolized = alert.attributes.symbolize_keys
```
Comment (bf4) [inline test/cache_test.rb]: I can see the value of adding a test for symbolizing, but it looks like you're replacing the existing one?

**Comment by aldreth** [comment]
@bf4 Thanks for your comments. I'm off on holiday for 2 weeks now. I'll pick this up when I come back.

**Comment by bf4** [comment]
@aldreth would still love some help here

**Comment by bf4** [comment]
Mobile.... Sorry, missed :smile:

**Pull request closed without merging**